pontoon/contributors/static/js/profile.js

```diff
@@ -127,9 +136,171 @@ var Pontoon = (function (my) {
         });
       },
     },
+    profile: {
+      /*
+       * This method is heavily inspired by the Github-Contribution-Graph,
+       * authored by bachvtuan.
+       *
+       * @see {@link https://github.com/bachvtuan/Github-Contribution-Graph}
+       */
+      renderContributionGraph: function () {
+        const graph = $('#contribution-graph');
+        const contributions = graph.data('contributions');
+
+        // Set start date to 365 days before now
+        var startDate = new Date();
```

Comment:
Mixing `const` and `var` like this is weird. Just make them all `const` and `let`, as appropriate.